pgjdbc/src/test/java/org/postgresql/jdbc/NamedParametersTest.java

```diff
@@ -82,7 +82,9 @@ public void testHasNamedParameters() throws SQLException {
     try (PGPreparedStatement testStmt = con.prepareStatement(sql).unwrap(PGPreparedStatement.class)) {
       Assert.assertFalse(testStmt.hasParameterNames());
       final PSQLException psqlException = Assert.assertThrows(PSQLException.class, testStmt::getParameterNames);
-      Assert.assertEquals("The ParameterList was not created with named parameters.", psqlException.getMessage());
+      Assert.assertEquals("No parameter names are available, you need to call hasParameterNames "
+          + "to verify the presence of any names.\n"
+          + "Perhaps you need to enable support for named placeholders?", psqlException.getMessage());
```

Comment:
What do you think of referring to the property here so the user doesn't have to figure it out ? Same with line 535 ?